Makefile

```diff
@@ -0,0 +1,69 @@
+# Define variables for Docker image tags and paths
+BROKER_SERVICE_TAG = broker-service
+PROOF_SERVICE_TAG = proof-service
+FTP_SERVICE_TAG = ftp-service
+SERVERLESS_TAG = serverless
+
+BROKER_SERVICE_DOCKERFILE = packages/broker-service/Dockerfile
+PROOF_SERVICE_DOCKERFILE = packages/proof-service/Dockerfile
+FTP_SERVICE_DOCKERFILE = packages/ftp/Dockerfile
+SERVERLESS_DOCKERFILE = packages/serverless/Dockerfile
+
+BROKER_SERVICE_COMPOSE = packages/broker-service/docker-compose.yml
+PROOF_SERVICE_COMPOSE = packages/proof-service/docker-compose.yml
+FTP_COMPOSE = packages/ftp/docker-compose.yml
+SERVERLESS_COMPOSE = packages/serverless/docker-compose.yml
+MONGO_COMPOSE = general/database/docker-compose.yml
+
+# Build Docker images
+.PHONY: build
+build: build-broker build-proof build-ftp build-serverless
+
+build-broker:
+	docker build --no-cache -t $(BROKER_SERVICE_TAG) -f $(BROKER_SERVICE_DOCKERFILE) --platform=linux/amd64 .
```

Comment:
Can we set platform as variable like above?

Reply:
Okay that's fine.